normalized-cache/api/normalized-cache.klib.api

```diff
@@ -578,9 +578,6 @@ final object com.apollographql.cache.normalized.api/EmptyMetadataGenerator : com
     final fun metadataForObject(kotlin/Any?, com.apollographql.cache.normalized.api/MetadataGeneratorContext): kotlin.collections/Map<kotlin/String, kotlin/Any?> // com.apollographql.cache.normalized.api/EmptyMetadataGenerator.metadataForObject|metadataForObject(kotlin.Any?;com.apollographql.cache.normalized.api.MetadataGeneratorContext){}[0]
 }
 
-final const val com.apollographql.cache.normalized/VERSION // com.apollographql.cache.normalized/VERSION|{}VERSION[0]
-    final fun <get-VERSION>(): kotlin/String // com.apollographql.cache.normalized/VERSION.<get-VERSION>|<get-VERSION>(){}[0]
-
```

Comment:
Interesting that removing the const is a breaking change on native. No `@JvmField` there. I think it's OK though 👍